src/pages/myinfo/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import _ from 'lodash';
+import { basicWrap, flexAlignCenter } from 'styles/containers';
+import Profile, { Animals, IconSize } from 'components/profile';
+import color from 'styles/colors';
+import InputBox from 'components/input-box';
+import Button, { TextButton as TextButtonCommon } from 'components/button';
+
+const OPTIONS = [
+  Animals.Hamster,
+  Animals.Puppy,
+  Animals.Rabbit,
+  Animals.Unicorn,
+  Animals.Fox,
+  Animals.Bear,
+  Animals.Panda,
+  Animals.Tiger
+];
+
+const IconSelector = styled.div`
+  padding: 23px 0 19px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Icons = styled.div`
+  width: 188px;
+  height: 96px;
+  margin-top: 16px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const ProfileWrap = styled.div<{ selected: boolean }>`
+  background-color: ${({ selected }) => (selected ? color.primary : null)};
+  margin-bottom: 8px;
+`;
+
+const ButtonWrap = styled.div`
+  margin-top: 32px;
+`;
+
+const TextButton = styled(TextButtonCommon)`
+  color: ${color.grayscale.gray03};
+`;
+
+const Divider = styled.div`
+  width: 1px;
+  height: 16px;
+  background-color: ${color.grayscale.gray05};
+  margin: 0 16px;
+`;
+
+export default function Myinfo() {
+  const [selected, setSelected] = useState(Animals.Hamster);
+  const [nickname, setNickname] = useState('주예');
+
+  const handleChange = _.debounce((value: string) => {
+    setNickname(value);
+    console.log(nickname);
+  }, 500);
+
+  return (
+    <div css={basicWrap}>
+      <IconSelector>
+        <Profile type={selected} iconSize={IconSize.XL} />
+        <Icons>
+          {OPTIONS.map((option) => (
+            <ProfileWrap selected={selected === option} onClick={() => setSelected(option)}>
+              <Profile type={option} />
+            </ProfileWrap>
+          ))}
+        </Icons>
+      </IconSelector>
+      <InputBox label="이름" note="최소 2자 최대 8자 입력가능해요." defaultValue="주예" onChangeInput={handleChange} />
+      <ButtonWrap>
+        <Button label="저장" />
+      </ButtonWrap>
+      <div
+        css={[
+          flexAlignCenter,
+          css`
+            margin-top: 57px;
+          `
+        ]}
+      >
+        <TextButton label="회원탈퇴" />
```

Comment:
Minor)
개인적으로 버튼 라벨은 children에 그대로 넘기는게 더 직관적인거같아유! (일반 html button tag처럼)
```suggestion
        <TextButton>회원탈퇴</TextButton>
```

Reply:
위 방식이 더 직관적이긴 하네요..! 반영해보겠습니당!

Reply:
37924c1 커밋에 반영되었습니당!